src/components/pages/ParentHome/ParentSidebar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Menu } from 'antd';
+import 'antd/dist/antd.css';
+
+//const { SubMenu } = Menu; if submenu is needed in the sidebar
+
+function ParentSidebar() {
+  const handleClick = e => {
+    console.log('click ', e);
+  };
+
+  return (
+    <Menu
+      onClick={handleClick}
+      style={{ width: 256 }}
+      defaultSelectedKeys={['1']}
+      defaultOpenKeys={['sub1']}
+      mode="inline"
+    >
+      <Menu.Item key="1">Overview</Menu.Item>
```

Comment:
Can we make the key the title of the menu item? Is that possible? Or does it have to a unique num?